api/filter_dimension_options_test.go

```diff
@@ -720,7 +720,7 @@ func TestSuccessfulGetFilterBlueprintDimensionOption(t *testing.T) {
 		w := httptest.NewRecorder()
 		api := routes(host, mux.NewRouter(), &mocks.DataStore{}, &mocks.FilterJob{}, &mocks.DatasetAPI{}, previewMock, enablePrivateEndpoints, downloadServiceURL, downloadServiceToken, mockAuditor)
 		api.router.ServeHTTP(w, r)
-		So(w.Code, ShouldEqual, http.StatusNoContent)
+		So(w.Code, ShouldEqual, http.StatusOK)
```

Comment:
As the status code has changed and there is now a body being returned, you will need to update specs

Reply:
done.